R/AddTestToPkg.R

```diff
@@ -0,0 +1,59 @@
+#################################################################################################### .
+# AddTestToPkg                                                                              ####
+#   Description - The function will add a new test function to the calculation package.
+#   This function is intended to be used when a file in the package was already created but
+#   a test file was not created.
+#################################################################################################### .
+#' @name  AddTestToPkg
```

Comment:
The use_test creates a bare bones file if I remember correctly where as this function uses a template with a header that we want for later data mining and can also include some basic code as examples, as comments so the test does not fail.  However, we could look at the use_template as this would replace the code that that covers.